src/test/java/hw3/ex2/DifferentElementsTest.java

```diff
@@ -0,0 +1,139 @@
+package hw3.ex2;
+
+import static hw3.DataClassTest.BASE_URL;
+import static hw3.DataClassTest.BROWSER_TITLE;
+import static hw3.DataClassTest.CHECKBOX;
+import static hw3.DataClassTest.CHECKBOX_LOG;
+import static hw3.DataClassTest.DIFFERENT_ELEMENTS_URL;
+import static hw3.DataClassTest.DROPDOWN;
+import static hw3.DataClassTest.DROPDOWN_LOG;
+import static hw3.DataClassTest.RADIO;
+import static hw3.DataClassTest.RADIO_LOG;
+
+import hw3.BaseClassTest;
+import hw3.pages.DifferentElementsPage;
+import hw3.pages.HomePage;
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.htmlelements.element.CheckBox;
+import ru.yandex.qatools.htmlelements.element.Radio;
+
+public class DifferentElementsTest extends BaseClassTest {
+
+    @Test
+    public void checkDifferentElementsTest() {
+
+        //1. Open test site by URL
+        HomePage homePage = new HomePage(webDriver, BASE_URL);
+
+        //2. Assert Browser title
+        softAssertions
+                .assertThat(homePage.getTitle())
+                .isEqualTo(BROWSER_TITLE);
+
+        //3. Perform login
+        homePage.getHeader().signIn(
+                properties.getString("login"),
+                properties.getString("pass")
+        );
+
+        //4. Assert User name in the left-top side of screen that user is loggined
+
+        softAssertions
+                .assertThat(homePage.getHeader().isUsernameDisplayed())
+                .isTrue();
+
+        softAssertions
+                .assertThat(homePage.getHeader().getUsername())
+                .isEqualTo(properties.getString("username"));
+
+        //5. Open through the header menu Service -> Different Elements Page
+        DifferentElementsPage differentElementsPage =
+                homePage.getHeader().clickDifferentElements();
+
+        softAssertions
+                .assertThat(differentElementsPage.getUrl())
+                .isEqualTo(DIFFERENT_ELEMENTS_URL);
+
+        //6. Select checkboxes
+        CheckBox checkBoxWater =
+                differentElementsPage
+                .getDifferentElements()
+                .getCheckbox(CHECKBOX.get(0));
+
+        CheckBox checkBoxWind =
+                differentElementsPage
+                .getDifferentElements()
+                .getCheckbox(CHECKBOX.get(2));
+
+        softAssertions
+                .assertThat(checkBoxWater)
+                .isNotNull();
+        softAssertions
+                .assertThat(checkBoxWind)
+                .isNotNull();
+
+        checkBoxWater.click();
+        checkBoxWind.click();
```

Comment:
Let's move page interactions to page objects
```java
        checkBoxWater.click();
        checkBoxWind.click();
```
So that in case we need to modify this actions in future (add some waits or change click implementation) we can simply modify page object class, and not all the tests where these actions are used

Reply:
Done